### ISSUE TITLE
Livesplit won't open

### ISSUE THREAD

**Issue by AWP82** [LiveSplit/LiveSplit#2056]
My livesplit suddenly won't open for some reason. It was working fine last night, but now when I click on it, nothing happens. It doesn't show up in task manager, and running as admin doesn't work, nor did redownloading it. All I can think of that may have caused this is a windows update this morning but that's a wild guess. Can anyone help me?

**Comment by Derpspace** [comment]
I had the same issue and found that turning off your antivirus fixes it.
Let me know if this helps.
https://github.com/LiveSplit/LiveSplit/issues/1753

**Comment by AWP82** [comment]
Nope. Just turned off my avast and windows firewall. It still won't open.

**Comment by AWP82** [comment]
> I had the same issue and found that turning off your antivirus fixes it.
> Let me know if this helps.
> #1753

Nope. Just turned off my avast and windows firewall. It still won't open.

**Comment by Mechinsam** [comment]
same problem with me. But when i try LiveSplit 1.6.9 it works

**Comment by thisis2838** [comment]
Is there any error messages from LiveSplit in Event Viewer?

**Comment by Mechinsam** [comment]
> Are there any error messages from LiveSplit in Event Viewer?

no, it just refuses to open for me

**Comment by jaydarschool** [comment]
Wait a minute, I may have an idea.
The `README.md` file has a section titled "Compiling".
It says that "To compile LiveSplit, you need one of these versions of Visual Studio:

- Visual Studio 2017 Community Edition
- Visual Studio 2017[...]"

Try installing one of those.

**Comment by just-ero** [comment]
Does this issue persist in the most recent stable version of LiveSplit?
